modules/lightning_features/lightning_core/lightning_core.module

```diff
@@ -81,6 +110,50 @@ function lightning_core_disable_buttons(array $element) {
 }
 
 /**
+ * Implements hook_form_alter().
+ */
+function lightning_core_form_alter(array &$form, FormStateInterface $form_state) {
+  $form_object = $form_state->getFormObject();
+
+  if ($form_object instanceof EntityFormInterface) {
+    $entity = $form_object->getEntity();
+
+    if ($entity instanceof EntityDisplayModeInterface || $entity instanceof RoleInterface) {
+      $form['description'] = [
+        '#type' => 'textarea',
```

Comment:
I decided to leave it as a textarea, since it hard to say how verbose these explanations may need to be...but made it dramatically smaller and wrapped it in a P tag when it's displayed.